### DISCUSSION TITLE
cusdis doesn't work in my blog based on hugo

### DISCUSSION THREAD

**Discussion by jmwyf** [djyde/cusdis#240] [Q&A] [answered]
I add cusdis comments.html in blog project layouts/partials/comments.html, but it doesn't work.
[blog code](https://github.com/jmwyf/jmwyf.github.io) is here. what's the possible reason？

**Comment by djyde** [comment]
Hi. What does it mean "doesn't work"? Is the comment widget displayed?

**Reply by jmwyf** [reply]
@djyde Comment widget does not display

**Reply by RoneoOrg** [reply]
I've been able to [set it up with Hugo](https://discourse.gohugo.io/t/free-and-open-source-comments-for-hugo/32940), please point precisely to the revelant code/commit AND live website if you want a review

**Reply by ghost** [reply, marked as answer]
> I've been able to [set it up with Hugo](https://discourse.gohugo.io/t/free-and-open-source-comments-for-hugo/32940), please point precisely to the revelant code/commit AND live website if you want a review

This appears to be a Congo theme issue, have resolved it for me there - hoping originator sees.
